internal/p/p.go

```diff
@@ -45,13 +54,14 @@ func NewCommand() *cobra.Command {
 
 	cmd.Flags().String("device-id", "", "device ID from 'spotify device list'")
 	cmd.Flags().String("playlist", "", "playlist name from 'spotify playlist list'")
+	cmd.Flags().String("album", "", "album name that you wish to play'")
 
 	return cmd
 }
 
-func p(api internal.APIInterface, query, contextQuery, deviceID string) (string, error) {
+func p(api internal.APIInterface, query, contextQuery, queryType, deviceID string) (string, error) {
```

Comment:
Let's pass all of the args and flags in here so we can test this more easily! (Anything outside of this function can't be unit tested)
```suggestion
func p(api internal.APIInterface, track, album, playlist, deviceID string) (string, error) {
```